heartbeat/docs/getting-started.asciidoc

```diff
@@ -198,88 +87,73 @@ was started. Heartbeat adds the `@every` keyword to the syntax provided by the
+It does not install {beatname_uc} dashboards.  Heartbeat dashboards and 
+installation steps are available in the
+https://github.com/elastic/uptime-contrib[uptime-contrib] github repository.
+
+[TIP]
+=====
+A connection to {es} (or {ess}) is required to set up the initial
+environment. If you're using a different output, such as {ls}, see
+<<load-template-manually>>.
+=====
+
+[float]
+[[start]]
 === Step 5: Start Heartbeat
 
-Start Heartbeat by issuing the appropriate command for your platform. If you
-are accessing a secured Elasticsearch cluster, make sure you've configured
-credentials as described in <<{beatname_lc}-configuration>>.
-
-NOTE: If you use an init.d script to start Heartbeat on deb or rpm, you can't
-specify command line flags (see <<command-line-options>>). To specify flags,
-start Heartbeat in the foreground.
-
-*deb and rpm:*
-
-["source","sh",subs="attributes"]
-----------------------------------------------------------------------
-sudo service {beatname_lc}-elastic start
-----------------------------------------------------------------------
-
-*mac and linux:*
+Before starting {beatname_uc}, modify the user credentials in
++{beatname_lc}.yml+ and specify a user who is
+<<privileges-to-publish-events,authorized to publish events>>.
 
-["source","sh",subs="attributes"]
-----------------------------------------------------------------------
-sudo chown root heartbeat.yml <1>
-sudo ./heartbeat -e
-----------------------------------------------------------------------
-<1> You'll be running Heartbeat as root, so you need to change ownership of the
-configuration file, or run Heartbeat with `--strict.perms=false` specified. See
-{beats-ref}/config-file-permissions.html[Config File Ownership and Permissions]
-in the _Beats Platform Reference_.
+To start {beatname_uc}, run:
 
 :requires-sudo:
-include::{libbeat-dir}/shared-brew-run.asciidoc[]
+include::{libbeat-dir}/tab-widgets/start-widget.asciidoc[]
 :requires-sudo!:
 
-*win:*
+Heartbeat is now ready to check the status of your services and send
+events to your defined output.
 
-["source","sh",subs="attributes"]
-----------------------------------------------------------------------
-PS C:\Program Files\Heartbeat> Start-Service heartbeat
-----------------------------------------------------------------------
+[float]
+[[view-data]]
+=== Step 6: View your data in {kib}
 
-By default, Windows log files are stored in +C:\ProgramData\heartbeat\Logs+.
+{beatname_uc} comes with pre-built {kib} dashboards and UIs for visualizing the
+status of your services. The dashboards are available in the
+https://github.com/elastic/uptime-contrib[uptime-contrib] github repository.
 
-Heartbeat is now ready to check the status of your services and send
-events to your defined output.
+If you loaded the dashboards earlier, open them now.
 
-[[view-kibana-dashboards]]
-=== Step 6: View the sample Kibana dashboards
+include::{libbeat-dir}/shared/opendashboards.asciidoc[tag=open-dashboards]
```

Comment:
> If you loaded the dashboards earlier, open them now.
>
> To open the dashboards:

Could we simplify to `If you loaded the dashboards earlier, open them now:`

Reply:
The more I think about this, the less I like my suggestion

Reply:
I think I like you suggestions, but will need to look in context.

Reply:
Hmm...Heartbeat is an outlier from the other Beats because the dashboards are not loaded when you run the setup command. (That's why the style here is a bit different.) 

I'm going to leave as-is for now because I can't think of a way to get around this problem without adding conditional coding, which I don't want to do.

Reply:
Sounds good to me